onadata/apps/logger/data_migration/decisioner.py

```diff
@@ -1,4 +1,9 @@
 
+NEW_FIELDS_KEY = 'new_fields'
+RM_FIELDS_KEY = 'removed_fields'
+MOD_FIELDS_KEY = 'modified_fields'
```

Comment:
those constants should probably be a part of MigrationDecisioner

Reply:
Done